ppr-ui/src/views/newMhrRegistration/MhrReviewConfirm.vue

```diff
@@ -155,19 +155,20 @@
 
 <script lang="ts">
 /* eslint-disable no-unused-vars */
-import { Component, Vue } from 'vue-property-decorator'
+import { Component, Mixins, Vue } from 'vue-property-decorator'
 import { HomeSections } from '@/components/mhrRegistration/YourHome'
 import { HomeCertificationOptions, RouteNames } from '@/enums'
 import { Getter } from 'vuex-class'
 import { MhrRegistrationDescriptionIF } from '@/interfaces'
+import { DateMixin } from '@/mixins'
 /* eslint-enable no-unused-vars */
 
 @Component({
   components: {
     HomeSections
   }
 })
-export default class MhrReviewConfirm extends Vue {
+export default class MhrReviewConfirm extends Mixins(DateMixin) {
```

Comment:
On the Date composable note, to me, it looks more of a unit than composable. Because we are really extending the functionality of a component, but rather format the dates. Just an observation.

Reply:
Not sure i follow

Reply:
I think global stateless functionality is the exact definition of what a composable is, irregardless of what the functionality does. 
As long as it's common and stateless, i believe it would make sense to make it a 'composable'.

Reply:
Had a typo there... looks like a util method. I thought composables are for stateful logic. It works so I'm good!

Reply:
Yes! Right, sorry, stateful 👍

Reply:
Is there a change required here?

Reply:
This could be put in formatter/util file, and we can use it across the app.

Reply:
@dimak1 Updated

Reply:
If you feel strongly about any further preferences, feel free to comment here and will tackle in a future commit 👍